packages/pf-codemods/index.js

```diff
@@ -35,7 +35,7 @@ function printResults(engine, results, format) {
   }
 
   // Don't show warnings
-  results.forEach(result => result.messages = result.messages.filter(message => message.severity === 2));
+  // results.forEach(result => result.messages = result.messages.filter(message => message.severity === 2));
```

Comment:
did you mean to leave this?

Reply:
No that was something from when I was trying out warnings. Will add it back.